torch/linalg/__init__.py

```diff
@@ -1626,6 +1626,18 @@
 
 The parameter :attr:`full_matrices` chooses between the full (default) and reduced SVD.
 
+The `driver` kwarg may be used in CUDA with a cuSOLVER backend to choose the algorithm used to compute the SVD.
```

Comment:
I realised after looking at the docs that this should be ``:attr:`driver` ``. Just add a fix for it if @IvanYashchuk has more comments about other things that need fixing.